iOS/Flows/People/PeopleCoordinator.swift

```diff
@@ -36,9 +43,230 @@ class PeopleCoordinator: PresentableCoordinator<[Person]> {
             self.peopleNavController.prepareForInvitations()
             
             Task {
-                await self.updateInvitation()
+                await self.inviteRemainingPeople()
             }.add(to: self.taskPool)
         }
     }
 }
 
+extension PeopleCoordinator {
+
+    /// Invites the next person in the array of people to invite and keeps doing so until there are no more people to invite.
+    /// Once there are no more people to invite, the finishInviting phase is called.
+    @MainActor
+    func inviteRemainingPeople() async {
+        // If there are no more people to invite, then finish the flow.
+        guard var person = self.peopleToInvite[safe: self.inviteIndex] else {
+            await self.finishInviting()
+            return
+        }
```

Comment:
A recursive function. The base case is when there aren't any more people to invite.